lib/CustomFieldsFilters/CustomFieldsFilters.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 
-import { CUSTOM_FIELDS } from '../constants';
+import { CUSTOM_FIELDS_FILTER } from '../constants/customFields';
```

Comment:
` from "../constants"` should work as well since you export all from `customFields.js` (`export * from './customFields';`) in `../constants/index.js`, right?